### PR TITLE
Hides hidden fields from json

### PR DESCRIPTION
**Pull request by peterbryntesson**
fix; hidden fields now doesn't show up in pubsub json for events
fix: added check so that namespace index isn't out-of-bounds

### CODE REVIEW

File: components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs [hunk 1]
```diff
@@ -1111,12 +1111,18 @@ public class MonitoredItemWrapper {
                         }
                     }
 
+                    List<SimpleAttributeOperand> addedSelectClauses = new List<SimpleAttributeOperand>();
```
Comment (simonjaeger) [inline components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs:1114]: Could you add a comment about this so it's clear what we are doing? And perhaps call it internalSelectClauses.

**Review by simonjaeger — APPROVED** [review]
LGTM

File: components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs (hunk 1 above)
Comment (peterbryntesson) [reply components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs:1114]: Good point. Fixed.

**Review by mregen — APPROVED** [review]
:shipit:

**Pull request merged**